gcloud-java-examples/src/main/java/com/google/cloud/examples/datastore/DatastoreExample.java

```diff
@@ -159,28 +198,98 @@ public void run(Transaction tx, Key userKey, String... args) {
         stBuilder.setLength(stBuilder.length() - 1);
         content = stBuilder.toString();
       }
-      IncompleteKey commentKey = IncompleteKey.builder(userKey, COMMENT_KIND).build();
-      FullEntity<IncompleteKey> comment = FullEntity.builder(commentKey)
-          .set("content", content)
-          .set("timestamp", DateTime.now())
+      return content;
+    }
+
+    @Override
+    protected String params() {
+      return "<comment>";
```

Comment:
If this takes multiple, wouldn't it make sense to return `String[]` and then have the thing that prints the help-text render that with `<>`'s and all that?